vdp/model/v1alpha/model_private_service.proto

```diff
@@ -12,7 +12,7 @@ import "vdp/model/v1alpha/model.proto";
 service ModelPrivateService {
   option (google.api.default_host) = "api.instill.tech";
 
-    // ========== Admin API ========== 
+    // ========== Admin API ==========
```

Comment:
We probably wanna name these APIs `Private API`. Need to change this in other backend protobuf comment too. (cc'd @xiaofei-du)